x-pack/legacy/plugins/canvas/server/routes/shareables.ts

```diff
@@ -25,15 +26,29 @@ export function shareableWorkpads(route: CoreSetup['http']['route']) {
   route({
     method: 'GET',
     path: API_ROUTE_SHAREABLE_RUNTIME,
+    // @ts-ignore Hapi types incorrect
+    config: {
```

Comment:
Per https://github.com/hapijs/hapi/blob/v17.5.3/API.md#route.options.files this is available on `options`, which is part of the type definition.
```suggestion
    options: {
```